### ISSUE TITLE
Can't ping anything 

### ISSUE THREAD

**Issue by 777Raim** [olegos2/mobox#157]
Hi there!
From `mobox` can't ping or install apps with internet connection (f.e. Union for Gothic).
Tried `ping google com` via `cmd`:

![Screenshot_20240126_081238_TermuxX11](https://github.com/olegos2/mobox/assets/30041514/59915006-53af-4f3b-809f-3cc2bfa4d29a)

It's seems that NDS works - `ping` gets right IP.

From `Termux` internet access and `ping` works as expected.

**Comment by Jacobw1oo** [comment]
Yep, it does that. I have not found a fix, but it has not affected my Internet/(DNS).

**Comment by n0201** [comment]
did you find a fix for it yet?
